index.html

```diff
@@ -324,13 +324,14 @@
           <dl>
             <dt>ffmpeg</dt><dd>starts the command</dd>
             <dt>-i <i>input_file</i></dt><dd>path and name of the input file</dd>
-            <dt>-write_id3v1 <i>1</i></dt><dd>Write ID3v1 tag. This will add metadata to the old MP3 format, assuming you’ve embedded metadata into the WAV file.</dd>
-            <dt>-id3v2_version <i>3</i></dt><dd>Write ID3v2 tag. This will add metadata to a newer MP3 format, assuming you’ve embedded metadata into the WAV file.</dd>
-            <dt>-dither_method <i>modified_e_weighted</i></dt><dd>Dither makes sure you don’t unnecessarily truncate the dynamic range of your audio.</dd>
+            <dt>-write_id3v1 <i>1</i></dt><dd>Write ID3v1 tag. This will add metadata to the "older" MP3 metadata format that's found at the head of the file, assuming you’ve embedded metadata into the WAV file.</dd>
+            <dt>-id3v2_version <i>3</i></dt><dd>Write ID3v2 tag. This will add metadata to the "newer" MP3 metadata format that's found at the tail of the file, assuming you’ve embedded metadata into the WAV file.</dd>
+            <dt>-dither_method <i>rectangular</i></dt><dd>Dither makes sure you don’t unnecessarily truncate the dynamic range of your audio.</dd>
```

Comment:
I'm not strong on the details of digital audio, so I wonder why you recommend this algorithm over `modified_e_weighted`? The ffmpeg docs have the list of possibilities, but don't explain what the differences or pros/cons of each are. So, it would be great if you're able to expand a bit here.